Riot/Modules/Common/Recents/RecentsViewController.m

```diff
@@ -64,6 +64,10 @@ @interface RecentsViewController () <CreateRoomCoordinatorBridgePresenterDelegat
     // when the user selects it.
     UISearchBar *tableSearchBar;
     
+    // Flag determining whether the view controller is ready to use (potentially shared) user indicators
+    // depending on whether the controller is visible or not
+    BOOL isUserIndicatorEnabled;
```

Comment:
Nothing mandatory. This name confused me a bit at first because I believed that you wanted to add the capability to enable / disable the activity indicator from other instance or subclasses. Why not not calling this property `isViewVisible` as it is exactly what you aim to check?

Reply:
Yea I guess that is more typical for cases like this and will be clearer.